<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiationToolboxDockWidgetBase</class>
 <widget class="QDockWidget" name="RadiationToolboxDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>315</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>244</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Radiation Toolbox (DEV)</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>9</number>
    </property>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLoadEdit">
       <attribute name="title">
        <string>Load and Edit</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>5</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="styleLabel">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Style:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="styleBox"/>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="styleButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QVBoxLayout" name="toolbarLayout"/>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStats">
       <attribute name="title">
        <string>Stats</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupStats">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
          </property>
          <property name="title">
           <string>ADER statistics</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPlot">
       <attribute name="title">
        <string>Plot</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupPlot">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
          </property>
          <property name="title">
           <string>ADER plot</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMaps">
       <attribute name="title">
        <string>Maps</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="margin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupOnlineMaps">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
          </property>
          <property name="title">
           <string>Online maps</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="2">
            <widget class="QPushButton" name="onlineMapsButton">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="onlineMapsCombo"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="onlineMapsLabel">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Map:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="margin">
         <number>5</number>
        </property>
        <item row="1" column="0">
         <widget class="QGroupBox" name="plotGroupBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
          </property>
          <property name="title">
           <string>Plot settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QGridLayout" name="plotGridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="plotStyleLabel">
               <property name="text">
                <string>Style:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="plotStyleCombo">
               <item>
                <property name="text">
                 <string>Lines</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Points</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="storageGroupBox">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 3px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
          </property>
          <property name="title">
           <string>Storage settings</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="storageLabel">
             <property name="text">
              <string>Format:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="storageCombo">
             <item>
              <property name="text">
               <string>SQLite</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GeoPackage</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Memory</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionImport">
   <property name="icon">
    <iconset>
     <normaloff>:/plugins/RadiationToolbox/icons/tool_import.png</normaloff>
     <normalon>:/plugins/RadiationToolbox/icons/tool_import.png</normalon>:/plugins/RadiationToolbox/icons/tool_import.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import radiation data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/plugins/RadiationToolbox/icons/tool_save.png</normaloff>:/plugins/RadiationToolbox/icons/tool_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save layer into new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset>
     <normaloff>:/plugins/RadiationToolbox/icons/tool_cut.png</normaloff>:/plugins/RadiationToolbox/icons/tool_cut.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select features to cut</string>
   </property>
  </action>
  <action name="actionDeselect">
   <property name="icon">
    <iconset>
     <normaloff>:/plugins/RadiationToolbox/icons/tool_cut_deselect.png</normaloff>:/plugins/RadiationToolbox/icons/tool_cut_deselect.png</iconset>
   </property>
   <property name="text">
    <string>Deselect</string>
   </property>
   <property name="toolTip">
    <string>Deselect features</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>:/plugins/RadiationToolbox/icons/tool_delete.png</normaloff>:/plugins/RadiationToolbox/icons/tool_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected features</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
